dearpygui_ext/logger.py

```diff
@@ -52,7 +56,7 @@ def auto_scroll(self, value):
 
     def _log(self, message, level):
 
-        if level < self.log_level:
+        if level <= self.log_level:
```

Comment:
This is a bugfix.

Reply:
Removed in next commit, let logging module send according to the setLevel